### PR TITLE
add fees description in transaction details

### PR DESCRIPTION
**Pull request by Epimodev**
This PR adds fees description in transaction details right panel:  
image

### CODE REVIEW

File: clients/banking/src/locales/en.json [hunk 1]
```diff
@@ -613,6 +613,15 @@
   "transaction.creditorName": "Creditor",
   "transaction.debtor": "Debtor",
   "transaction.executionDateTime": "Execution date",
+  "transaction.fees.description.bankingFee": "",
```
Comment (zoontek) [inline clients/banking/src/locales/en.json:616]: Is it normal that the translation is empty?

File: clients/banking/src/components/TransactionDetail.tsx [hunk 2]
```diff
@@ -138,6 +141,10 @@ export const TransactionDetail = ({ transaction, large }: Props) => {
         <Tile
           style={styles.tile}
           footer={match(transaction)
+            // BankingFee should never happened and is a default value, so there isn't any wording for it
```
Comment (zoontek) [inline clients/banking/src/components/TransactionDetail.tsx:144]: @Epimodev I don't get the meaning of the comment here 😶

Comment (Epimodev) [reply clients/banking/src/components/TransactionDetail.tsx:144]: yes I agree it's not clear. I just want to say this should not happen so we don't handle this case. It was approved by product team

Comment (zoontek) [reply clients/banking/src/components/TransactionDetail.tsx:144]: Ah, got it, `P.not("BankingFee")`

Comment (Epimodev) [reply clients/banking/src/components/TransactionDetail.tsx]: ℹ️ just changed comment here: 1139da1a166d093e5983098578b4f9da81a671c9

**Pull request merged**